examples/kubernetes/dynamic_provisioning/README.md

```diff
@@ -0,0 +1,49 @@
+## Dynamic Provisioning
+This example shows how to create a dynamically provisioned volume created through [EFS access points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html) and Persistent Volume Claim (PVC) and consume it from a pod.
+
+**Note**: this example requires Kubernetes v1.13+ and driver version >= 1.1.0.
```

Comment:
Also the kubernetes v ersion is not accurate anymore, I think we need 1.17+ because of the sidecars like the version of external-provisioner is only compatible with 1.17+ . Can you double check this and edit.

Reply:
Agreed. Updated